### DISCUSSION TITLE
Trying to import from milvus import Milvus, IndexType, MetricType getting error

### DISCUSSION THREAD

**Discussion by sanketp27** [milvus-io/milvus#26215] [Q&A and General discussion]
Can someone help me with below inputerror:
![image](https://github.com/milvus-io/milvus/assets/92051729/3fda8a7f-d30c-4dc4-a33c-db4a96b1d22d)

**Comment by yhmo** [comment]
It is better to start from an example of pymilvus: https://github.com/milvus-io/pymilvus/blob/2.2/examples/example.py

In fact, we don't need the IndexType, MetricType. Just input as string:
```
    index_param = {
        "index_type": "IVF_FLAT",
        "params": {"nlist": 256},
        "metric_type": "L2"}
    collection.create_index(filed_name, index_param)
```

Valid index types and metric types: https://milvus.io/docs/build_index.md
